src/style.css

```diff
@@ -0,0 +1,79 @@
+html {
+  font-size: 12px;
+  font-family: Roboto;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: 37px;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
+
+.logo {
+  position: absolute;
```

Comment:
dont stylize logo with absolute, you can align it with flex, just apply space-between to parent and align-items: center